internal/graph/storagewrapper.go

```diff
@@ -232,7 +253,7 @@ func (c *CachedDatastore) Read(
 	return c.newCachedIteratorByObjectRelation(ctx, "Read", store, iter, b.String(), tupleKey.GetObject(), tupleKey.GetRelation())
 }
 
-func findInCache(cache storage.InMemoryCache[any], store, key string, invalidEntityKeys []string) (*storage.TupleIteratorCacheEntry, bool) {
+func findInCache(cache storage.InMemoryCache[any], store, key string, invalidEntityKeys []string, logger logger.Logger) (*storage.TupleIteratorCacheEntry, bool) {
```

Comment:
Not for this PR, but I think `findInCache` should be made a private method within `cachedDatastore`.